src/bridge_px4/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ find_package(catkin REQUIRED COMPONENTS
     mavros_msgs
     mavlink
     std_msgs
-    message_generation
+#    message_generation
```

Comment:
I think we should remove all these comments if they are unneeded now, so we are keeping with a true clean up 😃

Reply:
I'm hesitant to remove it fully because I have a hunch that we will be using services in the future (e.g. to change setpoint modes from position to velocity). I'm pretty Davidian on how I do my CMakeLists too and I don't think it's currently worth the time to learn the proper way.

I'm going to carry on with the merge here because the repo in this branch's form will eliminate a lot of possible misunderstandings in this week's test. We can look into focused CMakeLists clean-up later (and maybe throw in a template somewhere).